lib/gzip_vas.c

```diff
@@ -167,12 +167,18 @@ int nx_function_end(nx_devp_t nxhandle)
 {
 	int rc = 0;
 
-	rc = munmap(nxhandle->paste_addr - 0x400, 4096);
-	if (rc < 0) {
-		fprintf(stderr, "munmap() failed, errno %d\n", errno);
-		return rc;
-	}
-	close(nxhandle->fd); /* see issue 164 comment */
+		rc = munmap(nxhandle->paste_addr - 0x400, 4096);
+		if (rc < 0) {
+			fprintf(stderr, "munmap() failed, errno %d\n", errno);
+			return rc;
+		}
+
+		rc = close(nxhandle->fd); /* see issue 164 comment */
+		if (rc < 0) {
+			fprintf(stderr, "close() failed, errno %d\n", errno);
+			return rc;
```

Comment:
Not an issue but this `return` could be removed.

Reply:
Fixed.